bson/_cbsonmodule.c

```diff
@@ -3227,11 +3227,19 @@ _cbson_exec(PyObject *m)
         INITERROR;
     }
 
+#if PY_VERSION_HEX >= 0x030D0000
+    if (PyModule_Add(m, "_C_API", c_api_object) < 0) {
+        Py_DECREF(c_api_object);
```

Comment:
We need to remove this decref, the new API always steals the reference: https://docs.python.org/3/c-api/module.html#c.PyModule_Add

Reply:
You removed the wrong line.

Reply:
😂

Reply:
Once again, you removed the wrong line.

Reply:
🤦 In 75233bb65a8db178931352f558b610fe9d6d1369 I only added two DECREFs … identical to the DECREFs in the `else` clause.  Are you saying we need to remove a DECREF from somewhere else, unrelated to this PR?

Reply:
We need to decref `m` here, not `c_api_object`.

Reply:
OK but that was already done in 9c768f5f25c4dd365cf374cee670ebbb50a87bb5 and you said that I removed the wrong one … 🤔

Reply:
No it was not done in https://github.com/mongodb/mongo-python-driver/commit/9c768f5f25c4dd365cf374cee670ebbb50a87bb5 .

That commit removes code in a completely unrelated code block.

Reply:
Thanks. FYI you should only resolve your own comments, not other people's. That way the reviewers can double check their comment is resolved.